lib/docker-compose.rb

```diff
@@ -62,36 +63,40 @@ def self.load_running_containers(compose)
   end
 
   def self.create_container(attributes)
+
+    service_config = ComposeUtils.parse_env_variables(attributes[1])
+
     ComposeContainer.new({
-      label: attributes[0],
-      name: attributes[1]['container_name'],
-      image: attributes[1]['image'],
-      build: attributes[1]['build'],
-      links: attributes[1]['links'],
-      ports: attributes[1]['ports'],
-      volumes: attributes[1]['volumes'],
-      command: attributes[1]['command'],
-      environment: attributes[1]['environment'],
-      labels: attributes[1]['labels']
-    })
+                             label: attributes[0],
```

Comment:
Could you please adjust this and the other indentation inconsistencies, please?
I need to add a lint support to the project, so sorry to ask that _a posteriori_